### PR TITLE
Fix device list down devices that have never been polled

### PR DESCRIPTION
**Pull request by murrant**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by PipoCanaja** [comment]
Oups ... makes sense ... Testing that :|

**Comment by PipoCanaja** [comment]
I would suggest this fix instead: 

```
'uptime' => $device->status ? Time::formatInterval($device->uptime, 'short') : ($device->last_polled ? Time::formatInterval($device->last_polled->diffInSeconds(), 'short') : 'Never'),

```

**Comment by PipoCanaja** [comment]
Longer, but the display is better in case last_polled is empty.

File: app/Http/Controllers/Table/DeviceController.php [hunk 1]
```diff
@@ -126,7 +126,7 @@ public function formatItem($device)
             'metrics' => $this->getMetrics($device),
             'hardware' => Rewrite::ciscoHardware($device),
             'os' => $this->getOsText($device),
-            'uptime' => Time::formatInterval($device->status ? $device->uptime : ($device->last_polled ? $device->last_polled->diffInSeconds() : null), 'short'),
+            'uptime' => $device->status ? Time::formatInterval($device->uptime, 'short') : ($device->last_polled ? Time::formatInterval($device->last_polled->diffInSeconds(), 'short') : 'Never'),
```
Comment (murrant) [inline app/Http/Controllers/Table/DeviceController.php]: `Uptime: Never` seems unclear.  Perhaps Never Polled

**Pull request merged**